models/reviews.model.js

```diff
@@ -7,4 +7,8 @@ exports.fetchReview = (reviewId) => {
                 if(data.rows.length === 0)return Promise.reject({msg: 'Review ID not found', status:404})
                 else return data.rows[0]
             })
+}
+exports.fetchAllReviews = () => {
+    return db.query(`SELECT CAST(COUNT(b.review_id) AS INTEGER) AS comment_count, a.* FROM reviews a FULL OUTER JOIN comments b ON b.review_id = a.review_id GROUP BY a.review_id ORDER BY a.created_at DESC;`)
```

Comment:
It looks like you are still returning all of the columns from review, including the body. We don't need the body of all the reviews, so this will need to be changed to select every column except for body.